src/libsyntax_ext/format.rs

```diff
@@ -46,13 +47,14 @@ struct Context<'a, 'b:'a> {
     /// Parsed argument expressions and the types that we've found so far for
     /// them.
```

Comment:
Can you update this documentation for what `arg_types` and `arg_unique_types` are?